lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEF.cs

```diff
@@ -138,7 +147,7 @@ private static IConfiguration GetRuntimeConfiguration(string name)
 
         public static void MainSimple(string[] args)
         {
-            var runtime = args.Length > 0 ? args[0] : Local;
+            var runtime = args.Length > 0 ? args[0] : AzureBatch;
 
```

Comment:
unnecessary change.

Reply:
Reverted.

---
In reply to: [200412162](https://github.com/sharathmalladi/reef/pull/6#discussion_r200412162) [](ancestors = 200412162)